trio/_core/tests/test_run.py

```diff
@@ -433,12 +433,12 @@ async def crasher():
                 # nursery block exited, all cancellations inside the
                 # nursery block continue propagating to reach the
                 # outer scope.
-                assert len(multi_exc.exceptions) == 5
+                assert len(multi_exc.exceptions) == 4
                 summary = {}
                 for exc in multi_exc.exceptions:
                     summary.setdefault(type(exc), 0)
                     summary[type(exc)] += 1
-                assert summary == {_core.Cancelled: 4, KeyError: 1}
+                assert summary == {_core.Cancelled: 3, KeyError: 1}
```

Comment:
Clarifying the cancelled count in a comment might be clearer here for those unfamiliar with the recent change?

Something like *we only receive cancelleds from subtasks not from the nursery itself*?